frontend/src/app/forms/forms.page.ts

```diff
@@ -0,0 +1,175 @@
+/*
+@Author
+Bharathwaj Ravi
+
+Add modifiers under @Modifiers
+@Modifiers
+
+@Purpose
+  - This page handles the basic operation of tracking progress, navigation, close menu, continue button,
+   event handlers for child to parent communication and parent to child data down.
+*/
+
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { NavController } from '@ionic/angular';
+import { Subscription } from 'rxjs';
+
+import { BasicInfo } from '../interfaces/basic-info';
+
+import { FormControl, FormGroup } from './../common/forms/forms';
+
+@Component({
+  selector: 'app-forms',
+  templateUrl: './forms.page.html',
+  styleUrls: ['./forms.page.scss'],
+})
+export class FormsPage implements OnInit, OnDestroy {
+  /*
+  @Usage this array holds the object with   information required for different views.
```

Comment:
I saw it in many places, but will only write it down here: Where is this "@Usage"/"@Whatever" notation from? I have never seen it on any TS project so far.

I would highly recommend adding comments like so:

``` TS
/**
 * This is a comment...
 */
sideMenuList = ...
```


The reason: Having the two asterisk at the beginning (`/**`) will allow us to use the  Intellisense (or whatever it is called) feature of VSCode. So, whenever you hover "this.sideMenuList", you will get a popup with your message.

Please update this in all places where you added a comment right above a method or attribute or class. The only time where you don't add two asterisk to a comment is when it doesn't belong to and class/attribute/method/... in particular.

Reply:
Oh, and if this `@Usage` style is nothing you took from a TS guideline, then please remove these "annotations". They are pretty uncommon for TS projects (except for stuff like `@param`).

Reply:
I have used these kind of custom comments in my previous projects in JS. I will remove it if it is not TS way of doing it.

Reply:
There are a few ones that are commonly used (like, as mentioned already, the `@param` annotation). But other than that, I would suggest using the comments starting with two asterisks (`/**`) and not using any custom annotations.